plugins/module_utils/aci.py

```diff
@@ -661,7 +661,8 @@ def _construct_url_1(self, obj):
         elif mo is None:
             # Query for all objects of the module's class (filter by properties)
             self.path = 'api/class/{0}.json'.format(obj_class)
-            self.update_qs({'query-target-filter': self.build_filter(obj_class, obj_filter)})
+            if obj_filter is not None:
```

Comment:
do we also need to check this condition in method _construct_url_2, _construct_url_3 and _construct_url_4?
for example, line 691, 695 and so on.

Reply:
We can add this condition when we see the usage of that function without this parameter. It was the case only for this one right now.